### DISCUSSION TITLE
Sample configuration file

### DISCUSSION THREAD

**Discussion by b23prodtm** [elarivie/atom-python-linters#10] [Ideas]
A configuration sample file is needed for [pydocstyle](https://buildmedia.readthedocs.org/media/pdf/pydocstyle/latest/pydocstyle.pdf)

**Comment by elarivie** [comment]
Hello @b23prodtm, effectively providing a sample setup.cfg file is a good Idea which I was already planning to add (I will probably add a link to the README)

But in the meantime if you want a sample configuration file see issue https://github.com/elarivie/atom-python-linters/issues/8#issuecomment-733404314

In this issue refered above I did include a setup.cfg file which I am using in a project


And find below a sample from the sample setup.cfg for the chunk about pydocstyle:
```
[pydocstyle]
inherit = false
ignore = D401,D203,D212
```
